model/jpa/src/main/resources/META-INF/orm.xml

```diff
@@ -60,11 +60,22 @@ http://java.sun.com/xml/ns/persistence/orm_2_0.xsd"
             </id>
         </attributes>
     </entity>
-    <entity class="iOSVariant" access="FIELD">
+    <entity class="iOSVariant">
         <discriminator-value>ios</discriminator-value>
         <attributes>
-            <basic name="certificate">
-                <lob/>
+            <many-to-one name="certificate" fetch="LAZY">
```

Comment:
Looks like this is actually causing the issue @sebastienblanc identified here: https://gist.github.com/sebastienblanc/77651f87529b44c3ab10